src/onegov/org/forms/newsletter.py

```diff
@@ -212,3 +220,102 @@ def recipient(self):
                     .one()
 
         return NewsletterSendFormWithRecipients
+
+
+class NewsletterSubscriberImportForm(Form):
+
+    dry_run = BooleanField(
+        label=_("Dry Run"),
+        description=_("Do not actually import the newsletter subscribers"),
+        default=False
+    )
+
+    file = UploadField(
+        label=_("Import"),
+        validators=[
+            DataRequired(),
+            WhitelistedMimeType({
+                'application/excel',
+                'application/vnd.ms-excel',
+                (
+                    'application/'
+                    'vnd.openxmlformats-officedocument.spreadsheetml.sheet'
+                ),
+                'application/vnd.ms-office',
+                'application/octet-stream',
+                'application/zip',
+                'text/csv',
+                'text/plain',
+            }),
+            FileSizeLimit(10 * 1024 * 1024)
+        ],
+        render_kw=dict(force_simple=True)
+    )
+
+    @property
+    def headers(self):
+        return {
+            'address': self.request.translate(_("Address")),
+        }
+
+    def run_export(self):
+        recipients = RecipientCollection(self.request.session)
+        headers = self.headers
+
+        def get(recipient, attribute):
+            result = (getattr(recipient, attribute, None))
```

Comment:
One pair of brackets to much, also use '' instead of None or else the next line might fail...